kubernetes/helm/caddyingresscontroller/templates/clusterrole.yaml

```diff
@@ -20,9 +20,9 @@ rules:
       - pods
       - nodes
       - routes
-      - extensions
```

Comment:
we'll still need `extensions/v1beta1` to see `Ingress` for pre-v1.14 versions of Kubernetes - is there any reason to _not_ support pre-v1.14?

Reply:
IBM and Google also deprecated v1.16